code/modules/multiz/zmimic/mimic_turf.dm

```diff
@@ -49,10 +53,7 @@
 		CRASH("Attempt to enable Z-mimic on already-enabled turf!")
 	shadower = new(src)
 	SSzcopy.openspace_turfs += 1
-	var/turf/under = GET_TURF_BELOW(src)
-	if (under)
-		below = under
-		below.above = src
+	GET_TURF_BELOW(src)
```

Comment:
does this do anything??

Reply:
Get turf below will automatically cache the turf that it finds underneath itself, it just initialises the value in it